src/rust/ide/enso-protocol/src/language_server/types.rs

```diff
@@ -342,6 +336,15 @@ pub struct CapabilityRegistration {
     pub register_options: RegisterOptions
 }
 
+impl CapabilityRegistration {
+    /// Create "text/canEdit" capability for path
+    pub fn create_can_edit(path:Path) -> Option<Self> {
```

Comment:
I'd consider including `text` as part of this method name. Currently this does not collide (the exeuction context is fortunately "modified" rather than "edited") but still...
Though yes, I know that in some other places we similarly don't follow, so this change might be bad for consistency or troublesome to fully push through.

So writing this just for consideration.